[35.001993179] 2017-12-06 22:46:03,784 - INFO - launcher: Starting sanity checks
[36.002159119] 2017-12-06 22:46:03,785 - INFO - launcher: Checking for python 3.5+
[36.002159119] 2017-12-06 22:46:03,785 - INFO - launcher: Checking console encoding
[36.002159119] 2017-12-06 22:46:03,785 - INFO - launcher: Setting console encoding to UTF-8
[37.002325058] 2017-12-06 22:46:03,786 - INFO - launcher: Ensuring we're in the right folder
[38.002252579] 2017-12-06 22:46:03,787 - INFO - launcher: Adding local bins/ folder to path
[39.002180099] 2017-12-06 22:46:03,788 - INFO - launcher: Checks passed.
[39.002180099] 2017-12-06 22:46:03,788 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[506.028890610] launcher-ERROR: Syntax error (this is a bug, not your fault)
Traceback (most recent call last):
  File "run.py", line 344, in main
    from musicbot import MusicBot
  File "C:\Users\acer\Desktop\Andy\Discord bots\MusicBot\musicbot\__init__.py", line 74, in <module>
    from .bot import MusicBot
  File "C:\Users\acer\Desktop\Andy\Discord bots\MusicBot\musicbot\bot.py", line 165
    else:
        ^
IndentationError: unindent does not match any outer indentation level
[507.029056549] launcher-INFO: All done.
